pytorch_lightning/core/lightning.py

```diff
@@ -1738,49 +1738,25 @@ def unfreeze(self) -> None:
 
     def get_progress_bar_dict(self) -> Dict[str, Union[int, str]]:
         r"""
+        Note: This method was deprecated in v1.5 and will be removed in v1.7.
+        Please use the `trainer.progress_bar_callback.get_progress_bar_dict` instead.
+
```

Comment:
there's a specific doc convention for marking it as deprecated. you can see this as an example: https://github.com/PyTorchLightning/pytorch-lightning/blob/de57feff445744f431eaef29182b1bcbf58a1924/pytorch_lightning/core/lightning.py#L1685-L1687